app/controllers/products_controller.rb

```diff
@@ -0,0 +1,49 @@
+class ProductsController < ApplicationController
+  before_action :set_product, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @products = Product.all
+  end
+
+  def show
+  end
+
+  def edit
+  end
+  
+  def new
+    @product = Product.new
+  end
+
+  def create
+    @product = Product.new(product_params)
+    @product.image.attach(params[:product][:image])
```

Comment:
you can use product_params here to get the image

Reply:
Thanks!